### ISSUE TITLE
Probleme avec la sauvegarde de photo 

### ISSUE THREAD

**Issue by jocelyncardinal** [CAUCA-9-1-1/survip-mobile#278]
Exemple ...

Fonctionne pas pour l'image suivante ...

 ![Capture d’écran 2018-11-27 à 14.49.52.png](https://images.zenhubusercontent.com/5aeb25264b5806bc2bd1a294/8c98e0c7-3011-49ac-a09b-cd8e783e3524)

par contre celle-ci fonctionne ...

![Capture d’écran 2018-11-27 à 14.51.32.png](https://images.zenhubusercontent.com/5aeb25264b5806bc2bd1a294/2e0b7b68-d08a-464c-b3ef-06bc17959e55)

**Comment by jocelyncardinal** [comment]
Audrey, je vais t'envoyer les deux images par courriel pour test

**Comment by jocelyncardinal** [comment]
Je note aussi problème avec sauvegarde des annotations ...


![Capture d’écran 2018-11-27 à 15.03.51.png](https://images.zenhubusercontent.com/5aeb25264b5806bc2bd1a294/0b5d3ec9-7b07-423e-a3db-3f422130fff6)

**Comment by iamkinetic** [comment]
Ça fait ça à tout coup avec ces deux images?

**Comment by jocelyncardinal** [comment]
Oui, je viens de retester et présentement ca bug a tout coup sur mon Iphone pour la premiere image et ca fonctionne a tout coup pour la 2e. Toutefois, il est possible que j'aille utilisé cette image (la 1ere) par le passé et que ca avait fonctionné a ce moment la et c'est pour ca que tu en l'a vue dans des inspection.
Note que ca fonctionne pour TOUTES les images sur le télephone **Android** a Audrey (elle n'est pas capable de reproduire le problème. Ca l'air d'etre relié au Iphone
Pour l'inspection que tu as vue avec l'image No1, elle vient probablement de tests que Audrey a fait (ou que jai fait précédemment et que l'image a réussi a etre sauvegarder a ce moment la).
Je vais te transférer les deux images pour voir si vous pouvez reproduire le probleme sur vos Iphones de test

**Comment by AudreyLR** [comment]
Je confirme que ça fonctionne de mon côté sur Android, ainsi que sur simulateur iphone dans mon environnement de développement. Par contre, je n'ai pas de iPhone pour tester, alors je n'ai pas pu reproduire ce bug.

**Comment by iamkinetic** [comment]
@philipperobertgh Peux-tu voir si tu peux reproduire le problème avec un des iPhones du bureau?

**Comment by iamkinetic** [comment]
@JocelynCardinal Nous sommes en train de modifier la configuration du serveur de test, donc il se peut que tu aies des erreurs dans les prochaines minutes.

**Comment by iamkinetic** [comment]
Asp.net core a une limite par défaut de 30mb par requête.  NGInx a quant à lui une limite de 1mb par requête.  Notre théorie est que NGinx est configuré avec la limite par défaut ce qui fait que certaines photos ne passent pas parce qu'elles dépassent le 1mb.  Nous attendons que le tech qui s'occupe de ça soit libre pour confirmer le tout.

**Comment by iamkinetic** [comment]
@philipperobertgh  Il faudra réfléchir à la façon de mieux gérer ça pour empêcher l'utilisateur d'essayer de mettre des photos trop grosses et de gérer correctement l'erreur quand on essaie d'ajouter une photo et que ça ne passe pas au niveau de l'API.

**Comment by philipperobertgh** [comment]
Le redimensionnement de l'image à la selection sur le disque n'est pas soumis aux même resolution selon le téléphone, c'est pour cela que cette situation ne se passe actuellement que sur IPhone. Nous allons ajuster la limite de transfert concernant les images. et mettre une validation pour les usagers si les images qu'ils tentent d'envoyer sont considérées comme trop grosses.

**Comment by philipperobertgh** [comment]
Le service technique est actuellement en vérification du problème de blocage de transfert de fichier vers le serveur de test.

**Comment by philipperobertgh** [comment]
Le service technique a augmenter la limite réseau pour les fichier volumineux, tester avec la photo transmise par Jocelyn.
Tout est fonctionnel.

**Comment by iamkinetic** [comment]
On peut faire un point avec le mobile pour bloquer les trop gros fichiers et augmenter la résolution de l'image (pour passer de 600x480 à une plus grosse résolution).  On pourrait faire des tests pour trouver une configuration qui a du sens : on peut jouer avec la résolution et le % de qualité et le format (jpg, png).  On devrait vérifier si on pourrait enlever la limite de résolution, mais baisser la qualité à 80% pour trouver un juste milieu.

Actuellement on est à 800x680, JPEG, 100%.

Et on devrait aussi créer un point pour le site web pour s'assurer que les images qu'on y ajoute ait les même limitations que sur mobile.

**Comment by jocelyncardinal** [comment]
Je confirme que ça fonctionne bien maintenant sur mon iPhone 8 
Testé plusieurs grosses images dans plan d’intervention et anomalies avec succès . 

Good job!!!
